experiments/book-surprisal/ocr_surprisal.py

```
@@ -0,0 +1,68 @@
+    for row in csv.DictReader(f, delimiter="\t"):
+        if row["likely_ocr_artifact"]!="Yes": continue
+        for pnum in row["pages"].split(","):
+            n=f"image{int(pnum):05d}.txt"
+            if n not in pages: continue
+            pg=pages[n]["first_pass"]; sv=sorted(pg["surprisal"]); pct=sv[int(0.95*len(sv))]
+            h=word_hits(row["word"], pg["text"], pg["offsets"], pg["surprisal"], pct); art_hits+=h; arts+=1 if h else 0
+random.seed(7); rnd_hits=[]
+for n,pg in pages.items():
+    fp=pg["first_pass"]; sv=sorted(fp["surprisal"]); pct=sv[int(0.95*len(sv))]
+    words=[w for w in re.findall(r"[A-Za-z]{3,}", fp["text"])]
+    for w in random.sample(words, min(3,len(words))): rnd_hits+=word_hits(w, fp["text"], fp["offsets"], fp["surprisal"], pct)[:1]
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)